lib/integrations/calendar/components/AddExchangeIntegration.tsx

```diff
@@ -20,7 +20,8 @@ export function AddExchangeIntegrationModal(props: DialogProps) {
     defaultValues: {
       username: "",
       password: "",
-      url: process.env.EXCHANGE_DEFAULT_EWS_URL || "",
+      url:
+        process.env.EXCHANGE_DEFAULT_EWS_URL || "https://exchange.reutlingen-university.de/Ews/Exchange.asmx",
```

Comment:
this should definitely not be here

Reply:
Sorry, pushed to the wrong repo, should be fixed now